packages/react-components/src/styles/index.ts

```diff
@@ -174,7 +175,7 @@ export default createGlobalStyle<Props & ThemeProps>(({ theme, uiHighlight }: Pr
 
       .ui--Icon {
         background: transparent;
-        color: ${getHighlight(uiHighlight)};
+        color: ${getHighlight(uiAltHighlight)};
```

Comment:
Here it can change to -

```js
color: inherit;
color: ${getHighlight(uiHighlight)};
```

So you never lose the button.

Reply:
> Not convinced, at it stands, the colors are already way too complex.
> 
> For the slider there is an alternative solution as well as for the in-table buttons (this one maybe not as colorful).

understood, the in-table buttons renders as black, it looks acceptable 0,0